pkgs/applications/audio/puddletag/default.nix

```diff
@@ -11,7 +11,9 @@ python2Packages.buildPythonApplication rec {
     sha256 = "1g6wa91awy17z5b704yi9kfynnvfm9lkrvpfvwccscr1h8s3qmiz";
   };
 
-  sourceRoot = "source/source";
+  setSourceRoot = ''
+    sourceRoot=$(echo */source)
+  '';
 
```

Comment:
It's been suggested not to use "source" directly - https://github.com/NixOS/nixpkgs/pull/31106#issuecomment-341266441 - hence I changed it as per the comment.